frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -678,7 +677,7 @@ end
 
 function ReaderDictionary:onShowDictionaryLookup()
     self.dictionary_lookup_dialog = InputDialog:new{
-        title = _("Enter a word to look up"),
+        title = _("Enter word(s) to look up"),
```

Comment:
I don't like this change. It's very unnatural.

Reply:
I can look up `upside down` (2 words) in English-Russian dictionary and get results.

Reply:
True, but the *wording* is weird ;).

The same meaning might be able to come across another way, possibly something based on "search term", which is understood to mean word/words/phrase ;).

Reply:
"Query" is perfect, it is duplicated in the lookup results window.
`Enter search query` for title, ok-button remains `Look up`.

Reply:
> I can look up upside down (2 words) in English-Russian dictionary and get results.

The proper term is `lemma` but the better solution is `word or phrase`.